### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by Miserlou**
## Issue Number

https://github.com/AlexsLemonade/refinebio/issues/435

## Purpose/Implementation Notes

Various small fixes and changes for related issues.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Miserlou** [comment]
Related: https://github.com/AlexsLemonade/refinebio/issues/432

File: common/data_refinery_common/job_lookup.py [hunk 1]
```diff
@@ -122,6 +122,13 @@ def determine_processor_pipeline(sample_object: Sample) -> ProcessorPipeline:
     if not _is_platform_supported(sample_object.platform_accession_code):
         return ProcessorPipeline.NONE
 
+    # Optional explicit filetype checks
+    if original_file:
+        if original_file.filename[-3].upper() == "CEL":
+            return ProcessorPipeline.AFFY_TO_PCL 
+        if original_file.filename[-3].upper() == "TXT":
+            return ProcessorPipeline.NO_COP
```
Comment (kurtwheeler) [inline common/data_refinery_common/job_lookup.py]: Hey man, I don't love cops either but it'd be great if you could keep your opinions out of the code :P

However are you 100% sure that we won't have any raw data files with .txt extensions? This seem to introduce the opportunity for us to NO_OP raw data if it's ever mis-named.

File: common/data_refinery_common/models/models.py [hunk 2]
```diff
@@ -324,7 +324,7 @@ def save(self, *args, **kwargs):
         return super(ExperimentAnnotation, self).save(*args, **kwargs)
 
 class Pipeline(models.Model):
-    "Pipeline that is associated with a series of ComputationalResult records."""
+    """Pipeline that is associated with a series of ComputationalResult records."""
```
Comment (kurtwheeler) [inline common/data_refinery_common/models/models.py]: :scream: What was happening before this?!?

Comment (Miserlou) [reply common/data_refinery_common/models/models.py]: Fortunately, it was just executing the string `"Pipeline that is associated with a series of ComputationalResult records."` and then the empty string `""`.

Comment (kurtwheeler) [reply common/data_refinery_common/models/models.py]: Oh whew!

**Comment by Miserlou** [comment]
NO_COP -> NO_OP

**Pull request merged**